src/CalendarView.vue

```diff
@@ -51,7 +56,7 @@ export default {
       return new Date(this.year, this.month + 1, 0).getDate();
     },
     firstDay() {
-      return new Date(this.year, this.month, 1).getDay();
+      return new Date(this.year, this.month, 1).getDay() - this.firstDayOfWeek;
```

Comment:
May need to handle cases where `firstDayOfWeek` is negative

Reply:
Agreed. I created a new variable in the data section to have a validated version of firstDayOfWeek to use here and inject to the child.